pkg/kubectl/cmd/plugin/plugin.go

```diff
@@ -116,8 +116,8 @@ func (o *PluginListOptions) Run() error {
 	for _, dir := range uniquePathsList(o.PluginPaths) {
 		files, err := ioutil.ReadDir(dir)
 		if err != nil {
-			if _, ok := err.(*os.PathError); ok && strings.Contains(err.Error(), "no such file") {
-				klog.V(3).Infof("unable to find directory %q in your PATH. Skipping...", dir)
+			if _, ok := err.(*os.PathError); ok {
```

Comment:
```
if pathErr, ok := err.(*os.PathError); ok {

}
```

Reply:
Not needed, since all I need from the bug is its string.